test/test/jfr/JfrTests.java

```diff
@@ -78,14 +79,35 @@ public void parseMultiModeRecording(TestProcess p) throws Exception {
         Assert.isGreater(eventsCount.get("jdk.ObjectAllocationInNewTLAB"), 50);
     }
 
+    private static void waitTtspWarmupOutput(TestProcess p) throws Exception {
```

Comment:
This was initially in `TestProcess`, but I moved it here since at the moment this is only used by this class. However, there might be other occurrences following the same pattern which could benefit from the construct

Reply:
I agree it may be useful to have a generic method that waits for the target process to spin up, especially on macOS where launching new process may take a long time. But for a solution to be generic, ideally it should not rely on the application doing something special. I was thinking of using JDK Attach Mechanism (like `jstack`) to verify if the JVM is up and running the main class. Anyhow, this deserves a separate discussion, not for this PR.